internal/pkg/cli/task_run.go

```diff
@@ -256,8 +306,14 @@ func (o *runTaskOpts) Execute() error {
 }
 
 func (o *runTaskOpts) runTask() ([]string, error) {
-	// TODO: run tasks
-	return nil, nil
+	o.spinner.Start(fmt.Sprintf("Waiting for %s to be running.", o.groupName))
+	taskARNs, err := o.runner.Run()
+	if err != nil {
+		o.spinner.Stop(log.Serrorf("Failed to run %s.\n", o.groupName))
+		return nil, fmt.Errorf("run task %s: %w", o.groupName, err)
+	}
+	o.spinner.Stop(log.Ssuccessf("Task(s) %s is running.\n", o.groupName))
```

Comment:
You can leverage this library if you'd like: https://github.com/dustin/go-humanize#plurals we already import it :)